lldb/source/API/CMakeLists.txt

```diff
@@ -196,13 +196,15 @@ elseif (LLDB_EXPORT_ALL_SYMBOLS)
   MESSAGE("-- Symbols (liblldb): exporting all symbols from the lldb and lldb_private namespaces")
 
   # Pull out the various lldb libraries linked into liblldb, these will be used
-  # when looking for symbols to extract. We ignore plugin libraries here,
-  # because these symbols aren't publicly exposed.
+  # when looking for symbols to extract. We ignore most plugin libraries here,
+  # because we may expose more symbols than the DLL limit and these symbols
+  # aren't useful to expose.
   get_target_property(all_liblldb_libs liblldb LINK_LIBRARIES)
   set(lldb_libs "")
   foreach(lib ${all_liblldb_libs})
     if(TARGET ${lib} AND ${lib} MATCHES "^lldb" AND
-       NOT ${lib} MATCHES "^lldbPlugin")
+       (${lib} MATCHES "^lldbPluginSymbolFileDWARF" OR
+        NOT ${lib} MATCHES "^lldbPlugin"))
```

Comment:
I'd prefer if we didn't hardcode this here. We should introduce another CMake variable that collects the list of plugins we want to export from or have a file specify it.

Reply:
SGTM, will update!

Reply:
It might be useful to have this cake variable be a cmake regex just to keep it simple and flexible

Reply:
Nvm, your current version using a list seems better